### ISSUE TITLE
More flexible blocking

### ISSUE THREAD

**Issue by ursbraem** [kanru/max-tabs#5]
This helps me a lot avoiding the "I don't know which tabs I've got open and what's my name anyway" moment in the second part of a day.  Thanks!

Though, it's a little frustrating when a _blank link you click is simply blocked.

I see the following alternative scenarios:

- Allow users to choose maxtab setting to apply for a) cmd-t new tab b) cmd-click links c) _blank links
- Instead of simply blocking the link, ask with a prompt
- Or (luxury solution) prompt which tab should be used

**Comment by SavannahJohnston** [comment]
Yes, this is a must-have feature. There especially needs to be a change in the way it handles tabs sent from other devices. I can't tell you how many tabs I've lost because I sent them to desktop and they tried to open in a window that was already at its limit. The extension definitely needs a prompt instead of just blocking everything indiscriminately. (Ideally, if possible, it should route sent tabs to a new window.)
